src/Protocol/Protocol.h

```diff
@@ -56,9 +56,84 @@ class cProtocol
 
 	virtual ~cProtocol() {}
 
+	/** A list of all outgoing packets */
+	enum eOutgoingPackets
+	{
+		sendAttachEntity = 0,
```

Comment:
I think the convention around the rest of the code for enum names is the first letters of the enum name + the field name, so for example opSendAttachEntity.